src/panel.js

```diff
@@ -529,20 +529,50 @@ const buildAllowedSitesPanel = async(panelId) => {
 
 const removeSiteFromContainer = async () => {
   const activeHostname = await getActiveHostname();
-  await browser.runtime.sendMessage( {removeDomain: activeHostname} );
-  browser.tabs.reload();
-  window.close();
+  buildRemoveSitePanel(activeHostname);
```

Comment:
Added "remove site" interstitial that was already in use in the site listed panel: 

![image](https://user-images.githubusercontent.com/2692333/74677975-55c6a680-517f-11ea-9784-2d081ca28da9.png)